go/cmd/vtcombo/plugin_dbddl.go

```diff
@@ -49,6 +54,6 @@ func (plugin *DBDDL) DropDatabase(ctx context.Context, name string) error {
 
 func init() {
 	servenv.OnRun(func() {
-		engine.DBDDLRegister("vttest", &DBDDL{})
+		engine.DBDDLRegister("vttest", &DBDDL{createDbLock: &sync.Mutex{}})
```

Comment:
If you use a non-pointer to store the mutex, you don't actually need to initialize the mutex.

```suggestion
		engine.DBDDLRegister("vttest", &DBDDL{})
```